### PR TITLE
Add Flickr Image Block Pattern

### PR DESCRIPTION
**Pull request by shubuu97**

### CODE REVIEW

File: inc/classes/class-block-patterns.php [hunk 1]
```diff
@@ -59,6 +59,21 @@ public function register_block_patterns() {
 					'content' => $two_columns_content,
 				]
 			);
+
+			/**
+             * Flickr Image Pattern
```
Comment (imranhsayed) [inline inc/classes/class-block-patterns.php:64]: Please fix the indentation

File: inc/classes/class-block-patterns.php [hunk 2]
```diff
@@ -80,6 +95,7 @@ public function register_block_pattern_categories() {
 		$pattern_categories = [
 			'cover' => __( 'Cover', 'aquila' ),
 			'columns' => __( 'Columns', 'aquila' ),
+			'flickr_image' => __( 'Flickr Image', 'aquila' ),
```
Comment (imranhsayed) [inline inc/classes/class-block-patterns.php:98]: Should the slug be just 'aquila_image` and name `Aquila Image`. Then we can put image patterns under the same category.

**Review by imranhsayed — CHANGES_REQUESTED** [review]
Few updates.. this will again be for practice.

File: inc/classes/class-block-patterns.php (hunk 2 above)
Comment (shubuu97) [reply inc/classes/class-block-patterns.php:98]: Ok, Will make the changes

File: inc/classes/class-block-patterns.php (hunk 1 above)
Comment (shubuu97) [reply inc/classes/class-block-patterns.php:64]: Screenshot 2020-10-14 at 5 04 20 PM
It is correct in the code, it is only showing this in github

File: inc/classes/class-block-patterns.php (hunk 2 above)
Comment (shubuu97) [reply inc/classes/class-block-patterns.php]: Done

**Pull request closed without merging**